src/module.c

```diff
@@ -10233,6 +10233,13 @@ static uint64_t moduleEventVersions[] = {
  *     * `REDISMODULE_SUBEVENT_EVENTLOOP_BEFORE_SLEEP`
  *     * `REDISMODULE_SUBEVENT_EVENTLOOP_AFTER_SLEEP`
  *
+ * * RedisModule_Event_Config
+ *
+ *     Called when a configuration event happens
+ *     The following sub events are available:
+ *
+ *     * `REDISMODULE_SUBEVENT_CONFIG_CHANGE`
```

Comment:
please also document the struct (like in RedisModuleEvent_SwapDB)

Reply:
done